jax/_src/pallas/triton/lowering.py

```diff
@@ -2606,6 +2607,10 @@ def _ir_constant(v: object, t: ir.Type) -> ir.Value:
   raise NotImplementedError
 
 
+def _i_constant(v: int) -> ir.Value:
```

Comment:
The name seems overly general. iiuc, this is only for loop indices?

Reply:
Inlined into the call site.